src/alarm/alarm_manager.cc

```diff
@@ -54,7 +54,8 @@ int AlarmManager::ScheduleAlarm(const std::string& app_id,
   }
 
   int alarm_id = 0;
-  if (alarm->type() == AlarmInfo::ABSOLUTE) {
```

Comment:
I was hoping one call less is better and more understandable. If you want I can change it.